litmus-portal/frontend/src/views/Home/AgentConfiguredHome/index.tsx

```diff
@@ -138,21 +138,19 @@ const AgentConfiguredHome: React.FC<AgentConfiguredHomeProps> = ({
             </ButtonFilled>
           }
           link={
-            <Link
-              underline="none"
-              color="primary"
+            <TextButton
+              variant="highlight"
               onClick={() => {
-                tabs.changeWorkflowsTabs(2);
                 history.push({
-                  pathname: '/workflows',
+                  pathname: '/myhub/Chaos%20Hub',
```

Comment:
is that space....ideally ChaosHub should a be single word right?

Reply:
@arkajyotiMukherjee  The `ChaosHub` in general is a term but here the `Chaos Hub ` is the name of the particular hub hence the space is required.